src/libcore/mem.rs

```diff
@@ -84,7 +86,7 @@ pub use intrinsics::transmute;
 /// mem::forget(file);
 /// ```
 ///
-/// The swap function uses forget to good effect.
+/// The `mem::swap()` function uses forget to good effect:
```

Comment:
It seems to be weird to include parens here, especially since it actually takes args.